src/elements/form.ts

```diff
@@ -0,0 +1,31 @@
+import { LitElement, html } from 'lit'
+import { customElement, property } from 'lit/decorators.js'
+
+@customElement('hangman-form')
+export class HangmanForm extends LitElement {
+  inputLetter: string = ''
```

Comment:
is this local state needed because we can't access the input element inside the `_handleSubmit` function?

Reply:
I couldn't get the element out of the shadowRoot. That's why I chose a private property

Reply:
I couldn't get the element out of the shadowRoot. That's why I chose a private property

Reply:
ok that's what i thought, all good then